voices/pt-PT.csv

```diff
@@ -1,222 +1,456 @@
-"39","38","38","","SYSTEM","0038.wav"
-"40","39","39","","SYSTEM","0039.wav"
-"41","40","40","","SYSTEM","0040.wav"
-"42","41","41","","SYSTEM","0041.wav"
-"43","42","42","","SYSTEM","0042.wav"
-"44","43","43","","SYSTEM","0043.wav"
-"45","44","44","","SYSTEM","0044.wav"
-"46","45","45","","SYSTEM","0045.wav"
-"47","46","46","","SYSTEM","0046.wav"
-"48","47","47","","SYSTEM","0047.wav"
-"49","48","48","","SYSTEM","0048.wav"
-"50","49","49","","SYSTEM","0049.wav"
-"51","50","50","","SYSTEM","0050.wav"
-"52","51","51","","SYSTEM","0051.wav"
-"53","52","52","","SYSTEM","0052.wav"
-"54","53","53","","SYSTEM","0053.wav"
-"55","54","54","","SYSTEM","0054.wav"
-"56","55","55","","SYSTEM","0055.wav"
-"57","56","56","","SYSTEM","0056.wav"
-"58","57","57","","SYSTEM","0057.wav"
-"59","58","58","","SYSTEM","0058.wav"
-"60","59","59","","SYSTEM","0059.wav"
-"61","60","60","","SYSTEM","0060.wav"
-"62","61","61","","SYSTEM","0061.wav"
-"63","62","62","","SYSTEM","0062.wav"
-"64","63","63","","SYSTEM","0063.wav"
-"65","64","64","","SYSTEM","0064.wav"
-"66","65","65","","SYSTEM","0065.wav"
-"67","66","66","","SYSTEM","0066.wav"
-"68","67","67","","SYSTEM","0067.wav"
-"69","68","68","","SYSTEM","0068.wav"
-"70","69","69","","SYSTEM","0069.wav"
-"71","70","70","","SYSTEM","0070.wav"
-"72","71","71","","SYSTEM","0071.wav"
-"73","72","72","","SYSTEM","0072.wav"
-"74","73","73","","SYSTEM","0073.wav"
-"75","74","74","","SYSTEM","0074.wav"
-"76","75","75","","SYSTEM","0075.wav"
-"77","76","76","","SYSTEM","0076.wav"
-"78","77","77","","SYSTEM","0077.wav"
-"79","78","78","","SYSTEM","0078.wav"
-"80","79","79","","SYSTEM","0079.wav"
-"81","80","80","","SYSTEM","0080.wav"
-"82","81","81","","SYSTEM","0081.wav"
-"83","82","82","","SYSTEM","0082.wav"
-"84","83","83","","SYSTEM","0083.wav"
-"85","84","84","","SYSTEM","0084.wav"
-"86","85","85","","SYSTEM","0085.wav"
-"87","86","86","","SYSTEM","0086.wav"
-"88","87","87","","SYSTEM","0087.wav"
-"89","88","88","","SYSTEM","0088.wav"
-"90","89","89","","SYSTEM","0089.wav"
-"91","90","90","","SYSTEM","0090.wav"
-"92","91","91","","SYSTEM","0091.wav"
-"93","92","92","","SYSTEM","0092.wav"
-"94","93","93","","SYSTEM","0093.wav"
-"95","94","94","","SYSTEM","0094.wav"
-"96","95","95","","SYSTEM","0095.wav"
-"97","96","96","","SYSTEM","0096.wav"
-"98","97","97","","SYSTEM","0097.wav"
-"99","98","98","","SYSTEM","0098.wav"
-"100","99","99","","SYSTEM","0099.wav"
-"101","100","100","","SYSTEM","0100.wav"
-"102","cento","cento","","SYSTEM","0101.wav"
-"103","duzentos","duzentos","","SYSTEM","0102.wav"
-"104","trezentos","trezentos","","SYSTEM","0103.wav"
-"105","quatrocentos","quatrocentos","","SYSTEM","0104.wav"
-"106","quinhentos","quinhentos","","SYSTEM","0105.wav"
-"107","seiscentos","seiscentos","","SYSTEM","0106.wav"
-"108","setecentos","setecentos","","SYSTEM","0107.wav"
-"109","oitocentos","oitocentos","","SYSTEM","0108.wav"
-"110","novecentos","novecentos","","SYSTEM","0109.wav"
-"111","mil","mil","","SYSTEM","0110.wav"
```

Comment:
Unfortunately, it's not quite a simple as that (unless the language pack is completely screwed up) ... there look to be an extra 10 or so sounds for the PT voice pack for the SYSTEM level. As part of the future goal to try and get this all in one document, maybe add the extra ones below the EN rows? i.e. keep the row ids the same for all the matching lines, and have all the unique lines for PT afterwards. 

This may give some guidance as to why there is a discrepancy... 

https://github.com/EdgeTX/edgetx/blob/main/radio/src/translations/tts_pt.cpp#L24-L67